build-tools/xaprepare/xaprepare/ConfigAndData/Configurables.cs

```diff
@@ -242,6 +248,19 @@ static string GetNetcoreAppRuntimePath (Context ctx, string androidTarget)
 				);
 			}
 
+			static string GetCoreClrAppRuntimePath (Context ctx, string androidTarget)
+			{
+				// TODO: The nuget id and the ref package version are guesses atm, since the CoreCLR packages don't exist yet
+				Log.Instance.Todo ("The nuget id and the ref package version are guesses atm, since the CoreCLR packages don't exist yet");
+				return Path.Combine (
+					XAPackagesDir,
+					$"microsoft.netcore.app.runtime.coreclr.android-{androidTarget}",
```

Comment:
```suggestion
					$"microsoft.netcore.app.runtime.android-{androidTarget}",
```

It is also required to include these packages for download. See https://github.com/ivanpovazan/android/commit/681f2158883bd2b8d2415109e567cbc8d313885e